.buildkite/pipeline.yaml

```
@@ -303,20 +303,20 @@ steps:
               push: true
               build-args:
                 - NAME=${APP_NAME}
-      - label: ":rocket: Trigger core-deploy"
-        key: "trigger-deploy"
+      - trigger: "core-deploy"
+        depends_on: "docker-pr-build"
+        label: ":rocket: Trigger core-deploy"
```

this is very much a nit, but can you lower case `trigger` - all the other steps labels are not capitalized 
```suggestion
        label: ":rocket: trigger core-deploy"
```